contracts/apps/AragonApp.sol

```diff
@@ -47,7 +41,7 @@ contract AragonApp is AppStorage, Autopetrified, VaultRecoverable, ReentrancyGua
     *         Always returns false if the app hasn't been initialized yet.
     */
     function canPerform(address _sender, bytes32 _role, uint256[] _params) public view returns (bool) {
-        if (!hasInitialized()) {
+        if (!isCallEnabled()) {
             return false;
         }
```

Comment:
We can move the `address(linkedKernel) == address(0)` check to be above this, as `isCallEnabled()` will always fail if the kernel doesn't exist. It may also save a _tiny_ bit of gas to inline the two conditions with an `||` 🤷‍♂